package.json

```
@@ -32,7 +32,8 @@
     "build:profile": "rm -rf build && cp envConfig/local.env .env && IS_PROD_PROFILE=true NODE_ENV=production webpack",
     "build:live": "cp envConfig/live.env .env && NODE_ENV=production webpack",
     "build:live:debug": "rm -rf build && awk '{sub(/LOG_DIR=.+/,\"LOG_DIR='log'\")}1' envConfig/live.env > .env && NODE_ENV=production webpack",
-    "build:preview": "cp envConfig/preview.env .env && NODE_ENV=production webpack",
+    "build:preview1": "cp envConfig/preview1.env .env && NODE_ENV=production webpack",
```

```suggestion
    "build:preview": "cp envConfig/preview1.env .env && NODE_ENV=production webpack",
    "build:preview1": "cp envConfig/preview1.env .env && NODE_ENV=production webpack",
```
As in https://github.com/bbc/simorgh-infrastructure/pull/1863#discussion_r1871493111 we need to maintain backward compatibility as `TARGET_ENV` is [not configurable](https://github.com/bbc/simorgh-infrastructure/blob/651675c84c9e17a8d78793a16b4a7b0f45dd0846/packages/preview-deploy/lib/preview-deploy-stack.js#L84) yet. 

I appreciate this wasn't in the ticket, this detail was missed sorry